substrate/client/statement-store/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub const DEFAULT_MAX_TOTAL_STATEMENTS: usize = 4 * 1024 * 1024; // ~4 million
 /// The maximum amount of data the statement store can hold, regardless of the number of
 /// statements from which the data originates.
 pub const DEFAULT_MAX_TOTAL_SIZE: usize = 2 * 1024 * 1024 * 1024; // 2GiB
+/// The maximum size of a single statement in bytes.
+/// Accounts for the 1-byte vector length prefix when statements are gossiped as `Vec<Statement>`.
+pub const MAX_STATEMENT_SIZE: usize = sc_network_statement::config::MAX_STATEMENT_SIZE as usize - 1;
```

Comment:
Maybe we can do it the other way around and allow this as is and increase the `MAX_STATEMENT_NOTIFICATION_SIZE` value to what it is +1.

Reply:
I was thinking of just moving the `-1` from here to `+1` there but anyway it's not that important.